spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -260,7 +285,7 @@ public void testBinPackAfterPartitionChange() {
   }
 
   @Test
-  public void testBinPackWithDeletes() throws Exception {
```

Comment:
I'm assuming we just forgot to remove this throws some time in the past?

Reply:
yes, IntelliJ suggested during code analysis and I think we dont really throw here.